src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -204,4 +205,10 @@
       <xsd:pattern value="(%.+%|true|false)" />
     </xsd:restriction>
   </xsd:simpleType>
+
+  <xsd:simpleType name="boolean_or_star">
+    <xsd:restriction base="xsd:string">
+      <xsd:pattern value="(%.+%|true|false|\*)" />
```

Comment:
I see, it's to allow parameters. I'm pretty sure allowing dynamic values for autowiring should not be allowed and makes no sense after all. I don't see the point of it.